frontend/src/pages/cans/detail/getCfy.js

```diff
@@ -0,0 +1,18 @@
+import { setCfy } from "./canFiscalYearSlice";
+import ApplicationContext from "../../../applicationContext/ApplicationContext";
+
+export const getCfy = (id) => {
```

Comment:
In general, I would expand `cfy` to its full name.  I'd do that here in the frontend and the backend.

Reply:
Refactored